packages/app/src/systems/Core/utils/databaseVersioning.ts

```diff
@@ -124,4 +123,30 @@ export const applyDbVersioning = (db: Dexie) => {
         });
       }
     });
+
+  // DB VERSION 23
+  // Drop transactions table since we don't use that anymore
+  db.version(23).stores({
+    vaults: 'key',
+    accounts: '&address, &name',
+    networks: '&id, &url, &name, chainId',
+    connections: 'origin',
+    transactions: null,
+    assets: '&name, &symbol',
+    abis: '&contractId',
+    errors: '&id',
+  });
+
+  // DB VERSION 24
+  // Add cursors table for handling tx pagination
+  db.version(24).stores({
+    vaults: 'key',
+    accounts: '&address, &name',
+    networks: '&id, &url, &name, chainId',
+    connections: 'origin',
+    transactionsCursors: '++id, address, providerUrl, endCursor',
+    assets: '&name, &symbol',
+    abis: '&contractId',
+    errors: '&id',
+  });
```

Comment:
Can't we drop transactions table in one go?

Reply:
oh yeah, let me merge this.

Reply:
do we need to change from `transactions` to `transactionsCursor` ? we can do it using same approach used before for changing pkey of another table in a previous upgrade. Using `transactionsTemp` then mnoving back to `transactions`

Reply:
@LuizAsFight i did that just because of naming 
I’m storing only the cursors, not the transaction itself

Reply:
@arthurgeron @LuizAsFight 
https://github.com/FuelLabs/fuels-wallet/pull/1558/commits/77793398cff376ac9b7898d5f44aa363c89a1100